### PR TITLE
Update depdendencies

### PR DESCRIPTION
**Pull request by kewisch**
This moves all dependencies up to the latest version and makes a few adjustments to accommodate for changes. I haven't been able to test this in practice yet, do you have some sort of test bed you usually use?

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ const processIssues = async ({
 
       return issues.data;
     } catch (e) {
-      core.error(`Get issues for repo error: ${e.message}`);
+      if (e instanceof Error) {
+        core.error(`Get issues for repo error: ${e.message}`);
+      }
```
Comment (kewisch) [inline src/index.ts:39]: I don't know if there is a better way to do this in TS, it seems a bit extraneous for simply showing an error message.

**Comment by kewisch** [comment]
@imhoffd I'm sure you've been busy, would love to get these updated so I can complete the work on the other pull request. Any chance you could take a look?

**Comment by imhoffd** [comment]
Thanks for this @kewisch, I went through the changes myself. I wanted to update a few more things, too.

**Comment by imhoffd** [comment]
> I haven't been able to test this in practice yet, do you have some sort of test bed you usually use?

You can test changes in branches or forks by using `@imhoffd/needs-reply@branch` in your yml files

**Pull request closed without merging**